docs/Datadog.Trace/README.md

```diff
@@ -0,0 +1,238 @@
+
+```csharp
+using Datadog.Trace;
+
+var settings = TracerSettings.FromDefaultSources();
+
+// settings.AdoNetExcludedTypes.Add("MySql");      // <- Delete this line
+settings.Integrations["MySql"].Enabled = false; // <- Add this line
+```
+
+### `ElasticsearchNet5` integration ID has been removed
+
+In version 1.x, the integration ID for version 5.x of `Elasticsearch.Net` was `ElasticsearchNet5`, and the integration ID for version 6.x+ was `ElasticsearchNet`. In version 2.0, `ElasticsearchNet5` has been removed. Use `ElasticsearchNet` for all versions of `Elasticsearch.Net`.
+
+Replace usages of `ElasticsearchNet5` with `ElasticsearchNet`
+
+```csharp
+using Datadog.Trace;
+
+var settings = TracerSettings.FromDefaultSources();
+
+settings.Integrations["ElasticsearchNet5"].Enabled = false; // <- Delete this line
+settings.Integrations["ElasticsearchNet"].Enabled = false;  // <- Add this line
+```
+
+### Obsolete APIs have been removed
+
+The following deprecated APIs were removed.
+
+* `TracerSettings.DebugEnabled` was removed. Set the `DD_TRACE_DEBUG` environment variable to `1` to enable debug mode.
+* `Tags.ForceDrop` and `Tags.ForceKeep` were removed. Use `Tags.ManualDrop` and `Tags.ManualKeep` respectively instead.
+* `SpanTypes` associated with automatic instrumentation spans, such as `MongoDb` and `Redis`, were removed.
+* `Tags` associated with automatic instrumentation spans, such as `AmqpCommand` and `CosmosDbContainer`, were removed.
+* `Tracer.Create()` was removed. Use `Tracer.Configure()` instead.
+* `TracerSettings.AdoNetExcludedTypes` was removed. Use `TracerSettings.Integrations` to configure ADO.NET automatic instrumentation.
+* Various internal APIs not intended for public consumption were removed.
+
+In addition, some settings were marked obsolete in 2.0:
+
+* Environment variables `DD_TRACE_ANALYTICS_ENABLED`, `DD_TRACE_{0}_ANALYTICS_ENABLED`, and `DD_TRACE_{0}_ANALYTICS_SAMPLE_RATE` for controlling App Analytics are obsolete. App Analytics has been replaced with Tracing Without Limits. See [our documentation for details](https://docs.datadoghq.com/tracing/legacy_app_analytics/).
+* `TracerSettings.AnalyticsEnabled`, `IntegrationSettings.AnalyticsEnabled`, and `IntegrationSettings.AnalyticsSampleRate` were marked obsolete.
+* Environment variable `DD_TRACE_LOG_PATH` is deprecated. Use `DD_TRACE_LOG_DIRECTORY` instead.
+
+### Introduction of `ISpan` and `IScope`
+
+Version 2.0 makes the public `Scope` and `Span` classes internal. Instead, we now expose public `IScope` and `ISpan` interfaces and the `Tracer` API was updated accordingly. If you are currently using explicit types (instead of interring types with `var`), replace usages of `Scope` with `IScope` and `Span` with `ISpan`:
```

Comment:
```suggestion
Version 2.0 makes the public `Scope` and `Span` classes internal. Instead, we now expose public `IScope` and `ISpan` interfaces and the `Tracer` API was updated accordingly. If you are currently using explicit types (instead of inferring types with `var`), replace usages of `Scope` with `IScope` and `Span` with `ISpan`:
```

Reply:
Oops, thanks!